src/app/modules/project/overview/overview.component.ts

```diff
@@ -84,7 +84,6 @@ export class OverviewComponent implements OnInit, AfterViewInit {
       // We need the 1ms timeout to make sure the components are rendered, I don't know why angular doesn't do this but whatever
       setTimeout(() => {
         this.amountOfProjectsOnSinglePage = this.filterMenu.amountOfProjectsOnSinglePage;
-        this.projectList.createProjectModal(projectId);
```

Comment:
This function call needs to be here, it opens the modal if a `projectId` is set, I.E dex.software.com/projects/12

Reply:
I didn't realise that's what it did, I'll change it to refer to the modalutility too. Thanks!

Reply:
I fixed it in this last commit I think